scripts/jest/config.source.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+module.exports = {
+  modulePathIgnorePatterns: [
+    '<rootDir>/scripts/rollup/shims/',
+    '<rootDir>/scripts/bench/',
+  ],
+  transform: {
+    '.*': require.resolve('./preprocessor.js'),
+  },
+  setupFiles: [require.resolve('./setupEnvironment.js')],
+  setupTestFrameworkScriptFile: require.resolve('./setupTests.js'),
+  testRegex: '/__tests__/.*(\\.js|\\.coffee|[^d]\\.ts)$',
+  moduleFileExtensions: ['js', 'json', 'node', 'coffee', 'ts'],
```

Comment:
What is .node? I don't see it anywhere (other than .node.js)

Reply:
Dunno. Used to be there. Maybe because it's one of Jest defaults? I can remove.